app/server/src/main/scala/org/bitcoins/server/BitcoinSServerMain.scala

```diff
@@ -90,25 +93,68 @@ class BitcoinSServerMain(override val serverArgParser: ServerArgParser)(implicit
 
   def startBitcoinSBackend(): Future[Unit] = {
     val start = System.currentTimeMillis()
-    if (nodeConf.peers.isEmpty) {
-      throw new IllegalArgumentException(
-        "No peers specified, unable to start node")
-    }
 
-    val peerSockets = {
-      nodeConf.peers.map(
-        NetworkUtil.parseInetSocketAddress(_, nodeConf.network.port)
-      )
+    val peerSocketsF = {
```

Comment:
this should probably be done inside the `Node`, we can probably change it to not take `peers` as a param now

Reply:
Could be done in a follow up PR as this might be a pretty large change